odl/trafos/__init__.py

```diff
@@ -33,3 +33,6 @@
 from . import wavelet
 from .wavelet import *
 __all__ += wavelet.__all__
+
+from .convolution import *
+__all__ += convolution.__all__
```

Comment:
Not sure about the location. Needs to be at least as "high" as `trafos`, though, since the Fourier implementation will depend on the FT.